claat/proto-renderer/testing-utils/general.go

```diff
@@ -0,0 +1,66 @@
+package testingUtils
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"reflect"
+	"runtime"
+	"testing"
+
+	"github.com/googlecodelabs/tools/third_party"
+)
+
+// unsupportedType is a dummy type used to showcase the failures of rendering
+// non-proto custom types since we take in "any" type as rendering input.
+type UnsupportedType struct{}
+
+// Simple proto constructor
+func NewDummyProto(in string) *tutorial.StylizedText {
+	return &tutorial.StylizedText{
+		Text: in,
+	}
+}
+
+// Function signature for output-format agnositic 'Render'
+type renderingFunc func(interface{}) (io.Reader, error)
+
+// Type for canonical i != o and !ok rendering template tests
+type RendererTestingBatch struct {
+	InProto interface{}
```

Comment:
privatize if possible

Reply:
again, not possible since using this type outside of this package